documentation/docs/test-suites/semantic-interoperability/cli/configuration.md

```diff
@@ -183,3 +186,73 @@ packages/
 ```
 
 You have now successfully configured the Tier 2 test suite to test your credentials against the core UNTP data model.
+
+## Setting Up Configuration for the New Data Model
+
+The Tier 2 test suite currently supports the existing UNTP data model. To test credentials with a new data model that includes the following schemas: Conformity Credential, Digital Facility Record, Traceability Events, and Product Passport, you can generate a configuration file.
+
+Running the command `yarn run untp config` will create a configuration file containing the default values, structured as follows:
+
+```json
+{
+  "credentials": [
+    {
+      "type": "conformityCredential",
+      "version": "v0.5.0",
+      "dataPath": "",
+      "url": ""
+    },
+    {
+      "type": "digitalFacilityRecord",
+      "version": "v0.5.0",
+      "dataPath": "",
+      "url": ""
+    },
+    {
+      "type": "traceabilityEvent",
+      "version": "v0.5.0",
+      "dataPath": "",
+      "url": ""
+    },
+    {
+      "type": "productPassport",
+      "version": "v0.5.0",
+      "dataPath": "",
+      "url": ""
+    }
+  ]
+}
+```
+
+The default version used will be the latest version of the schema. You can update the version to match the specific version of the schema you want to test against.
+
+### You can add additional schemas in two ways:
+
+#### 1. Add Directly to the Configuration File:
+
+You can add an object representing the schema type and version directly to the credentials array in the configuration file, for example:
+
+```json
+{
+  "type": "objectEvent",
+  "version": "v0.3.10",
+  "dataPath": "",
+  "url": ""
+}
+```
+
+#### 2. Add to Default Model in Code:
+
+You can also add the name of the schema type to the `untpDefaultModel` in the packages/untp-test-suite/src/interfaces/utils/credentials.ts file as follows:
+
+```typescript
+export const untpDefaultModel = [
+  'conformityCredential',
+  'digitalFacilityRecord',
+  'traceabilityEvent',
+  'productPassport',
+  'objectEvent',
+];
+```
```

Comment:
Should we update these event names to reflect the latest changes in the schema folder structure?
Since I noticed you have updated the schema folder names by adding a 'digital' prefix.